### PR TITLE
QF-912 - Further Accessibility Fixes

### PR DESCRIPTION
**Pull request by PeterDavidWilliams**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Web/Views/Dashboard/Index.cshtml [hunk 1]
```diff
@@ -10,6 +10,10 @@
 <main class="govuk-main-wrapper " id="main-content" role="main">
     <div class="govuk-grid-row">
 
+        <h1 class="govuk-heading-xl">
+            Apprenticeship assessment service
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Views/Dashboard/Index.cshtml:14]: Did this come from Content or UX design - I'm surprised to see this

Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Web/Views/Dashboard/Index.cshtml:14]: Need to talk with Victor about a screen design change only for Accessibility which seems wrong and to James to see if the same text as this which is in the header bar could become the h1 which is needed for the accessibility

Comment (PeterDavidWilliams) [reply src/SFA.DAS.AssessorService.Web/Views/Dashboard/Index.cshtml]: Update with new hidden content per conversation with James King, Jazzy and Victor

**Review by ChrisJWoodcock — APPROVED** [review]

**Comment by ChrisJWoodcock** [comment]
This has been merged and combined into a QF-503 branch for easier testing.

**Pull request closed without merging**